terraform/deployable/health_monitor_lambda.py

```diff
@@ -0,0 +1,64 @@
+""" Health monitoring lambda """
+import json
+import logging
+import os
+import boto3
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
I wonder if we should pass the log level in from the env so we can have different log level between test and production.